### DISCUSSION TITLE
single-cell-training/2022.7/day2/clustering/

### DISCUSSION THREAD

**Discussion by giscus** [sib-swiss/single-cell-training#36] [Q&A]
# single-cell-training/2022.7/day2/clustering/

https://sib-swiss.github.io/single-cell-training/2022.7/day2/clustering/

**Comment by avishai987** [comment]
How do I change number of neighbors in FindClusters? Which parameter is it?

**Reply by taniawyss** [reply]
You can change this in the FindNeighbors() function used before FindClusters(). It is the k.param argument. The default is 20, there is a suggestion here to use the same for the UMAP and clustering: [https://github.com/satijalab/seurat/issues/4717](https://github.com/satijalab/seurat/issues/4717)
